typhon/src/main.rs

```diff
@@ -45,9 +48,12 @@ async fn main() -> std::io::Result<()> {
     tracing::subscriber::set_global_default(tracing_subscriber::FmtSubscriber::new()).unwrap();
 
     let args = Args::parse();
-    std::env::set_var("HASHED_PASSWORD", args.hashed_password);
 
     // Initialization
+    let mut password_file = File::open(args.password)?;
+    let mut password = String::new();
+    password_file.read_to_string(&mut password)?;
+    std::env::set_var("PASSWORD", password);
```

Comment:
Oh, I never saw that: we are setting an env var for the current process which in turn is read by typhon-core?
We should do something different there!

But that's orthogonal to this PR. I filed issue #7.